main/src/libraries/wrdac/include/wrdac/subsystems/subSystem_KARMA.h

```diff
@@ -98,6 +110,36 @@ namespace wysiwyd {
             yarp::sig::Vector applySafetyMargins(const yarp::sig::Vector& in);
 
             /**
+             * @brief toolAttach (KARMA): wrapper for tool-attach of KARMA, can be used to choose the arm for actions with KARMA
+             * @param armType: string value of "left" or "right" arm
+             * @param dimTool: Vector of 3 dimension of tool in the frame attached to the hand
+             */
+            bool toolAttach(const std::string &armType, const yarp::sig::Vector &dimTool);
```

Comment:
Can we rename this method to `selectArm()`, and get rid of the `dimTool` parameter which is never used within WYSIWYD?